pallets/weights/src/pallet_staking.rs

```diff
@@ -8,210 +8,210 @@ use polymesh_runtime_common::{RocksDbWeight as DbWeight, Weight};
     }
     fn set_payee() -> Weight {
-        (29_842_000 as Weight)
+        (25_417_000 as Weight)
             .saturating_add(DbWeight::get().reads(1 as Weight))
             .saturating_add(DbWeight::get().writes(1 as Weight))
     }
     fn set_controller() -> Weight {
-        (70_035_000 as Weight)
+        (61_089_000 as Weight)
             .saturating_add(DbWeight::get().reads(3 as Weight))
             .saturating_add(DbWeight::get().writes(3 as Weight))
     }
+    // WARNING! Some components were not used: ["c"]
     fn set_validator_count() -> Weight {
-        (1_941_000 as Weight).saturating_add(DbWeight::get().writes(1 as Weight))
+        (4_176_000 as Weight).saturating_add(DbWeight::get().writes(1 as Weight))
     }
     fn force_no_eras() -> Weight {
-        (1_873_000 as Weight).saturating_add(DbWeight::get().writes(1 as Weight))
+        (5_140_000 as Weight).saturating_add(DbWeight::get().writes(1 as Weight))
     }
     fn force_new_era() -> Weight {
-        (1_854_000 as Weight).saturating_add(DbWeight::get().writes(1 as Weight))
+        (5_336_000 as Weight).saturating_add(DbWeight::get().writes(1 as Weight))
     }
     fn force_new_era_always() -> Weight {
-        (1_854_000 as Weight).saturating_add(DbWeight::get().writes(1 as Weight))
+        (4_615_000 as Weight).saturating_add(DbWeight::get().writes(1 as Weight))
     }
     fn set_invulnerables(v: u32) -> Weight {
-        (2_109_000 as Weight)
-            .saturating_add((11_000 as Weight).saturating_mul(v as Weight))
+        (5_008_000 as Weight)
+            .saturating_add((9_000 as Weight).saturating_mul(v as Weight))
             .saturating_add(DbWeight::get().writes(1 as Weight))
     }
     fn force_unstake(s: u32) -> Weight {
-        (50_902_000 as Weight)
-            .saturating_add((1_932_000 as Weight).saturating_mul(s as Weight))
+        (112_144_000 as Weight)
+            .saturating_add((6_127_000 as Weight).saturating_mul(s as Weight))
             .saturating_add(DbWeight::get().reads(4 as Weight))
-            .saturating_add(DbWeight::get().writes(7 as Weight))
+            .saturating_add(DbWeight::get().writes(8 as Weight))
             .saturating_add(DbWeight::get().writes((1 as Weight).saturating_mul(s as Weight)))
     }
     fn cancel_deferred_slash(s: u32) -> Weight {
-        (782_599_000 as Weight)
-            .saturating_add((1_743_000 as Weight).saturating_mul(s as Weight))
+        (8_079_708_000 as Weight)
+            .saturating_add((50_218_000 as Weight).saturating_mul(s as Weight))
             .saturating_add(DbWeight::get().reads(1 as Weight))
             .saturating_add(DbWeight::get().writes(1 as Weight))
     }
-    fn payout_stakers(n: u32) -> Weight {
-        (267_975_000 as Weight)
-            .saturating_add((123_000 as Weight).saturating_mul(n as Weight))
+    // WARNING! Some components were not used: ["n"]
+    fn payout_stakers(_: u32) -> Weight {
```

Comment:
Not sure if parameter `n` makes sense here.
If `MaxNominatorRewardedPerValidator` is updated by a Runtime update, we could also update the this file according to that.